### DISCUSSION TITLE
Does Feishin support the WebNowPlaying pluigin for apps like Rainmeter?

### DISCUSSION THREAD

**Discussion by 7eventy7** [jeffvli/feishin#692] [General]
Trying to make a rainmeter skin that shows various information from the currently playing song in Feishin. From my understanding the easiest way to do this is usually thru the WebNowPlaying plugin but I'm not sure if its integrated. If you know any other way I can pull information about the currently playing track for a script please let me know!!!

**Comment by iiPythonx** [comment]
The best method I can think of would be pulling directly from the [feishin websocket](https://github.com/jeffvli/feishin/blob/development/src/main/features/core/remote/index.ts#L332-L485). You're able to send it an authorization payload and it will send you back the current status (along with future status updates). If you were on Linux, you would be able to use the built-in MPRIS integration.

**Reply by 7eventy7** [reply]
![image](https://github.com/user-attachments/assets/5e1ec7dc-8de9-4b83-83a9-b35384d57359)
Got it working thanks so much!
